frontend/routes/Register.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <FormLayout>
+        <Card>
+            <form>
+                <h1>{{ $t("pages.register.main.title") }}</h1>
+                <TextInput id="fname" :placeholder="$t('pages.register.main.firstName')" fullWidth />
+                <TextInput id="lname" :placeholder="$t('pages.register.main.lastName')" fullWidth />
+                <TextInput id="email" :placeholder="$t('pages.register.main.email')" type="email" fullWidth />
+                <TextInput id="password" :placeholder="$t('pages.register.main.password')" type="password" fullWidth />
+                <TextInput id="confirm" :placeholder="$t('pages.register.main.confirmPassword')" type="password" fullWidth />
+                <Button class="center" :content="$t('pages.register.main.register')" type="submit"></Button>
+            </form>
+        </Card>
+        <Card>
```

Comment:
This card generally needs more spacing between elements. Can the contents of this card have the same spacing as the other card?

Reply:
By other card, I assume you are referring to the card above it. Unless I am mistaken, it already has the same spacing as that card.
![image](https://user-images.githubusercontent.com/31863372/148327625-14864ef4-8a2c-4509-9325-298be7fdbbc4.png)

Reply:
By 'other card', I am assuming you are referring to the card above it. If so, it appears to already have the same spacing as that one.

Reply:
I see it correctly now that I'm on my desktop. My apologies; I don't know why it was displaying differently for me at the time. I'm guessing it's a Parcel cache issue.